.idea/encodings.xml

```diff
@@ -3,6 +3,5 @@
   <component name="Encoding">
     <file url="file://$PROJECT_DIR$" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/src/main/kotlin" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/main/resources" charset="UTF-8" />
   </component>
 </project>
```

Comment:
First, I am not sure if this removed line affects the project at all so if some of you could test it that would be great.

Reply:
Do not commit this file, please ignore it.